subprojects/common/report-api/build.gradle.kts

```diff
@@ -0,0 +1,14 @@
+plugins {
```

Comment:
this module looks non-independent
In module `report-api` I expect to find some interface `Report` with contract
And anyone could create an impl module for that `Report`

What I actually see: bunch of models I don't know how to use

Reply:
Sure, just not in single PR;
There are a lot of weird dependencies i am trying to unravel right now )